css-two-column-layout/styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Karla, sans-serif;
+}
+
+#container {
```

Comment:
the container should have a set width, you also dont need justify content center here because the container is not a flex container